src/nexuscli/cli/__init__.py

```diff
@@ -8,6 +8,7 @@
   nexus3 (upload|up) <from_src> <to_repository> [--flatten] [--norecurse]
   nexus3 (download|dl) <from_repository> <to_dst> [--flatten] [--nocache]
   nexus3 (delete|del) <repository_path>
+  nexus3 version
```

Comment:
Could you please do this as an option instead of command? (i.e. `nexus3 --version`)

Reply:
done!